app/components/Transfers/configureDelegation/RemoveDelegationPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { MultiStepFormPageProps } from '~/components/MultiStepForm';
+import Button from '~/cross-app-components/Button';
+import { useCalcDelegatorCooldownUntil } from '~/utils/dataHooks';
+import { getFormattedDateString } from '~/utils/timeHelpers';
+import { AccountInfo } from '~/utils/types';
+import { withPendingDelegationChangeGuard } from './util';
+
+import styles from './DelegationPage.module.scss';
+
+interface RemoveDelegationPageProps
+    extends Pick<MultiStepFormPageProps<undefined>, 'onNext'> {
+    // eslint-disable-next-line react/no-unused-prop-types
+    accountInfo: AccountInfo | undefined; // This is needed for the pending delegation change guard HoC.
+}
+
+const RemoveDelegationPage = withPendingDelegationChangeGuard(
+    ({ onNext }: RemoveDelegationPageProps) => {
+        const cooldownUntil = useCalcDelegatorCooldownUntil();
+
+        return (
+            <>
+                <p className="flexChildFill">
+                    This will withdraw the delegated amount of the account.
```

Comment:
Pretty sure I just took it from the design.. Danielle will probably have an opinion on this when testing starts? Otherwise, we can get her to go through the flows.

Reply:
Hmm, I think it's weird, because it sounds like the money are removed from the account.

Reply:
I rephrased it to "return to the public balance of the account"